pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -31,7 +31,7 @@ makeWrapper() {
     local original="$1"
     local wrapper="$2"
     shift 2
-    local flagsBefore
+    local -a flagsBefore=()
```

Comment:
isn't it redundant to have `-a` and `=()`?

Reply:
Yeah. I wanted to get across "it's an array, and it's initialized to the empty array", rather than "just" declaring it with `-a` (never mind that in bash that arrays are always initialized).